component/sections/pricing/variant_c.js

```diff
@@ -87,14 +90,24 @@ function VariantC({ caption, title, description, plans }) {
                       {plans?.[0]?.primaryButton?.label && (
                         <a
                           className="block mt-6 w-full py-2 px-6 rounded-l-xl rounded-t-xl bg-webriq-blue hover:bg-webriq-darkblue text-white font-bold leading-loose transition duration-200"
-                          target={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "_blank" : null}
-                          rel={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "noopener noreferrer" : null}
+                          target={
+                            plans?.[0]?.primaryButton?.linkTarget ===
+                            "openLinkToNewTab"
+                              ? "_blank"
+                              : null
+                          }
+                          rel={
+                            plans?.[0]?.primaryButton?.linkTarget ===
```

Comment:
This **_rel_** attribute value was not updated.  Need to change this and as follows to
`plans?.[index]?.primaryButton?.linkTarget === "_blank" ? "noopener noreferrer" : null` .